Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -34,6 +34,7 @@ If you want to look directly into Java source code on a GUI, simply open your AP
 
 Android decompilers go one step further and attempt to convert Android bytecode back into Java source code, making it more human-readable. Fortunately, Java decompilers generally handle Android bytecode well. The above mentioned tools embed, and sometimes even combine, popular free decompilers such as:
 
+- [APKLab](https://github.com/APKLab/APKLab "APKLab")
```

Comment:
Could you please remove this line and include APKLab below as:

Alternatively you can use the [APKLab](0x08a-Testing-Tools.md#apklab) extension for Visual Studio Code or run [apkx](0x08a-Testing-Tools.md#apkx) on your APK or use the exported files from the previous tools to open the reversed source code on your preferred IDE.

Reply:
fixed: 86fd672